metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -431,9 +433,18 @@ class MetalsLanguageServer(
       () => userConfig,
       clientConfig
     )
+
+    stacktraceAnalyzer = new StacktraceAnalyzer(
+      workspace,
+      definitionProvider,
+      clientConfig.icons.findsuper,
+      clientConfig.doctorFormat() == DoctorFormat.Html
```

Comment:
this ideally should be a check for VSCode (or client that can execute commands inside html)

Reply:
🤔 I wonder if we are broadening the `doctorFormat` too much? Or maybe the name no longer fully captures what it is being used for. In the scenario, if a user has `http` enabled and they also have the doctor format being Html, will it open in the browser?

Reply:
replaced it with dedicated setting :+1: